products.json

```diff
@@ -375,6 +375,50 @@
                         ]
                     }
                 ]
+            },
+            {
+                "text": "Web & Cloud Solutions",
+                "state": {
+                    "opened": false
+                },
+                "children": [
+                    {
+                        "text": "Dräger Gas Detection Connect",
+                        "state": {
+                            "opened": false
+                        },
+                        "children": [
+                            {
+                                "text": "Frontend license agreements",
+                                "a_attr": {
+                                    "href": "license_agreements/GDC/GDC_frontend.pdf"
```

Comment:
BTW: Those .pdf files have all have »Empfänger (Firma)« in their title instead of a proper title.

Reply:
Will clarify tomorrow.